independent-projects/tools/platform-descriptor-api/src/main/java/io/quarkus/platform/tools/ConsoleMessageFormat.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.platform.tools;
+
+public final class ConsoleMessageFormat {
+
+    public static final String OK = "\u2705";
+    public static final String NOK = "\u274c";
+    public static final String NOOP = "\uD83D\uDC4D";
```

Comment:
Better make these attributes private otherwise any methods in this class are meaningless